packages/libs/src/utils/types.ts

```diff
@@ -162,6 +162,7 @@ export type TrackMetadata = {
   permalink: string
   audio_upload_id: Nullable<string>
   preview_start_seconds: Nullable<number>
+  placement_hosts: Nullable<string>
```

Comment:
I don't have context here, but should this be optional?

Reply:
definitely! the context here is this is an option for content owners who want to only have their content live on specific nodes. most uploaders won't care and will let the system do its default thing